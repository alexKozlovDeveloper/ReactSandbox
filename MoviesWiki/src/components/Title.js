import React, { Component } from "react";

class Title extends Component {
    render() {
        return (
            <div>
                <div>Title {this.props.name}</div>
            </div>
        );
    }
}

export default Title;