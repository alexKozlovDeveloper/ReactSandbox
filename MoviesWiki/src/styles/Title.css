.title{
    color: white;
    padding: 3px;
}