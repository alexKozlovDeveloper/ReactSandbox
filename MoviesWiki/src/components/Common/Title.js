import React, { Component } from "react";

import styles from "../../styles/Title.css"

const Title = ({ title }) => {
    return <div className={styles.title}>
        {title}
    </div>;
}

export default Title;