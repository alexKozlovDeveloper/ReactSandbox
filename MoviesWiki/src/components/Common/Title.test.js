import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import Title from "./Title";

configure({ adapter: new Adapter() });

test('Title rendering test', () => {

    var title = "someTitle";

    const component = shallow(<Title title={title} />);
    expect(shallowToJson(component)).toMatchSnapshot();
});

