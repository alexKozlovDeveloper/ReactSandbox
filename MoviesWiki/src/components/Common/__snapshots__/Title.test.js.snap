// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title rendering test 1`] = `
<div
  className="title"
>
  someTitle
</div>
`;
